phase2/kubeadm/configure-vm.sh

```diff
@@ -0,0 +1,26 @@
+# This is not meant to run on its own, but extends phase1/gce/configure-vm.sh
```

Comment:
Can we leave both of these in phase1/gce/ and rename them configure-vm-kubeadm.sh and configure-vm-ignition.sh?

we should avoid being cloud/platform specific in phase2/ code.